package.json

```diff
@@ -1,34 +1,8 @@
 {
-  "scripts": {
-    "clean": "rm -rf lib lib-es6",
-    "test": "jest",
-    "test:cov": "jest --config jest.config.ts.js --coverage",
-    "test:watch": "jest --config jest.config.ts.js --watch",
-    "prepare": "yarn clean && tsc && tsc --build tsconfig.es6.json",
-    "benchmark": "matcha benchmark.js"
-  },
-  "devDependencies": {
-    "@types/jest": "^23.3.1",
-    "@types/lodash": "^4.14.116",
-    "@types/node": "^10.7.1",
-    "jest": "^23.5.0",
-    "lodash": "^4.17.10",
-    "matcha": "^0.7.0",
```

Comment:
`jest` and `matcha` (benchmark runner) are replaced by `vitest`, which `vitest` also offers `vitest bench` command.